build/LegacyPackages/14.0/Microsoft.Build.Conversion.Core/Microsoft.Build.Conversion.Core.nuspec

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>$id$</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
```

Comment:
Our other nuget packages also point to that .Net Library license. I don't know which one we should point to, but I guess it should be the same for all our packages.

@Sarabeth-Jaffe-Microsoft